packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2416,10 +2434,13 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
             break;
           }
 
-          case RoapScenario.ANSWER:
+          case RoapScenario.ANSWER: {
```

Comment:
What about the ROAP offer and other cases here? We wouldn't require updating SDP in those cases

Reply:
I didn't think so. We would be sending out SDPs only on answers right? 

Do we send out SDP for any offers?

Reply:
Yes @mkesavan13 we send SDP for offer, offer request and answer